app/src/main/java/com/vero/tasky/auth/presentation/components/PasswordTextField.kt

```diff
@@ -0,0 +1,79 @@
+package com.vero.tasky.auth.presentation.components
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.*
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Visibility
+import androidx.compose.material.icons.filled.VisibilityOff
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.input.PasswordVisualTransformation
+import androidx.compose.ui.text.input.VisualTransformation
+import androidx.compose.ui.unit.dp
+import com.vero.tasky.ui.theme.onTextFieldHint
+import com.vero.tasky.ui.theme.onTextFieldIcon
+import com.vero.tasky.ui.theme.text
+import com.vero.tasky.ui.theme.textFieldBackground
+
+@Composable
+fun PasswordTextField(
+    hintRes: Int,
```

Comment:
Gonna mention it one last time, as to not spam the whole PR with it

@StringRes